README.md

```diff
@@ -141,6 +141,12 @@ The default port is `3000`
 You can then upload a local picture, i.e one from this repository in `test/data/`, or choose
 one of the `Image Samples` from the browser window.
 
+# Sample Output
+
+Here's a sample using an image of Tokyo at night:
+
+<img src="doc/source/images/SampleCitiesFromSpace.png" width="250">
```

Comment:
this is not checked in, and use markdown...

`![](doc/source/images/SampleCitiesFromSpace.png)`